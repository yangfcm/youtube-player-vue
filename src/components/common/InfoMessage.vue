<template>
  <div class="ui segment basic">
    <h3 class="ui center aligned header">
      <i class="info circle icon"></i>
      <slot></slot>
    </h3>
  </div>
</template>

<script>
import Blank from "./Blank";
export default {
  components: {
    appBlank: Blank,
  },
  props: ["message"],
};
</script>

<style scoped>
</style>