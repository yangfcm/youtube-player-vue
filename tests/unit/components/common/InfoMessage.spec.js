import { shallowMount } from "@vue/test-utils";
import InfoMessage from "@/components/common/InfoMessage";

test("InfoMessage component can render correctly", () => {
  const infoMessage = "Some infomation";
  const wrapper = shallowMount(InfoMessage, {
    slots: {
      default: infoMessage,
    },
  });
  expect(wrapper).toMatchSnapshot();
  expect(wrapper.find("h3").text()).toContain(infoMessage);
});
