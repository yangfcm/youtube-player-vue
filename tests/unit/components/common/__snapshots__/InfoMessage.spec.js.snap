// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoMessage component can render correctly 1`] = `
<div class="ui segment basic">
  <h3 class="ui center aligned header"><i class="info circle icon"></i> Some infomation</h3>
</div>
`;
